root/static/scripts/release-editor/fields.js

```diff
@@ -845,8 +852,118 @@ const validation = require('../edit/validation');
             $(element)
                 .prop("disabled", disabled)
                 .toggleClass("disabled-hint", disabled)
-                .attr("title", disabled ? i18n.l("This medium has one or more discids which prevent this information from being changed.") : "");
+                .attr("title", disabled ? l("This medium has one or more discids which prevent this information from being changed.") : "");
+        }
+    };
+
+
+    ko.bindingHandlers.artistCreditEditor = {
```

Comment:
This is how the release editor's knockout stuff renders the AC editor for each track. It's initiated like https://github.com/metabrainz/musicbrainz-server/pull/263/files#diff-6edb40ffe5e08874aaa80abd6c4d1045R192

In the `update` handler below (which runs whenever the track's artistCredit observable changes), we use React to render the AC editor into the current element.